Tests/L1Tests/tests/test_AVInput.cpp

```diff
@@ -75,6 +197,871 @@ TEST_F(AVInputTest, RegisteredMethods)
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setVRRSupport")));
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getVRRSupport")));
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getVRRFrameRate")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getInputDevices")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("writeEDID")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("readEDID")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getRawSPD")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getSPD")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setEdidVersion")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getEdidVersion")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getHdmiVersion")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setMixerLevels")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("startInput")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("stopInput")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setVideoRectangle")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getSupportedGameFeatures")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getGameFeatureStatus")));
+}
+
+TEST_F(AVInputInit, getInputDevices)
+{
+    EXPECT_CALL(*p_hdmiInputImplMock, getNumberOfInputs())
+        .WillOnce(::testing::Return(1));
+    EXPECT_CALL(*p_compositeInputImplMock, getNumberOfInputs())
+        .WillOnce(::testing::Return(1));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Invoke(connection, _T("getInputDevices"), _T("{}"), response));
+    EXPECT_EQ(response, string("{\"devices\":[{\"id\":0,\"connected\":false,\"locator\":\"hdmiin:\\/\\/localhost\\/deviceid\\/0\"},{\"id\":0,\"connected\":false,\"locator\":\"cvbsin:\\/\\/localhost\\/deviceid\\/0\"}],\"success\":true}"));
```

Comment:
In the API documentation, I don't see "success" in reponse. If it is part of the code, then we need to create a ticket to fix the documentation. Please note down these discrepancies for all APIs.